aea/test_tools/test_contract.py

```diff
@@ -144,19 +144,23 @@ def refill_from_faucet(
 
     @staticmethod
     def sign_send_confirm_receipt_transaction(
-        tx: JSONLike, ledger_api: LedgerApi, crypto: Crypto, sleep_time: float = 1.0
+        tx: JSONLike,
+        ledger_api: LedgerApi,
+        cryptos: List[Crypto],
+        sleep_time: float = 2.0,
```

Comment:
This is a breaking change to the API. It is also not clear why this should be done - it changes the meaning of the API! Probably add new functionality for the multi-signature case!